src/api/api.ts

```diff
@@ -0,0 +1,17 @@
+const BASE_URL = 'https://mate.academy/students-api';
+
+export const getTodos = () => {
+  return fetch(`${BASE_URL}/todos`)
+    .then(response => response.json());
+};
+
+export const getUser = async (userId: number) => {
+  try {
+    const response = await fetch(`${BASE_URL}/users/${userId}`);
+    const user = await response.json();
+
+    return user;
+  } catch {
+    return !Error;
```

Comment:
what is this? return new Error with error.message you've got from catch param

Reply:
or handle error inside component
https://mate-academy.github.io/fe-program/js/extra/fetch#handle-an-error-in-the-component